config/v16/profile_schemas/Internal.json

```diff
@@ -110,6 +110,13 @@
             ]
         },
         "UseTPM": {
```

Comment:
Is there a reason the key changed for 201 but not 16?

Reply:
good one, will change v201 back to original to allow for backwards compatibility